css-media-queries/at-media.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Media Queries - At Media</title>
+  <link rel="stylesheet" href="reset.css">
+  <style>
+
+    body {
+      min-height: 100vh;
+      background-color: palevioletred;
+    }
+
+    /* your code here 👇 */
+    @media screen and (min-width: px) and (max-width: 768px){
```

Comment:
That first media feature with no `px` measurement is unnecessary, and should be removed